src/features/page_view_event/aggregate/index.js

```diff
@@ -112,32 +113,22 @@ export class Aggregate extends FeatureBase {
         if (!entry.startTime || entry.startTime <= 0) return
 
         if (entry.name === 'first-paint') {
-          chunksForQueryString.push(param('fp', String(Math.floor(entry.startTime))))
+          queryParameters.fp = String(Math.floor(entry.startTime))
         } else if (entry.name === 'first-contentful-paint') {
-          chunksForQueryString.push(param('fcp', String(Math.floor(entry.startTime))))
+          queryParameters.fcp = String(Math.floor(entry.startTime))
         }
         paintMetrics[entry.name] = Math.floor(entry.startTime) // this is consumed by Spa module
       })
     } catch (e) {}
 
-    chunksForQueryString.push(param('xx', info.extra))
-    chunksForQueryString.push(param('ua', info.userAttributes))
-    chunksForQueryString.push(param('at', info.atts))
-
-    var customJsAttributes = stringify(info.jsAttributes)
-    chunksForQueryString.push(param('ja', customJsAttributes === '{}' ? null : customJsAttributes))
-
-    var queryString = fromArray(chunksForQueryString, agentRuntime.maxBytes)
-
-    // Capture bytes sent to RUM call endpoint (currently `1`) as a supportability metric. See metrics aggregator (on unload).
-    agentRuntime.bytesSent[protocol] = (agentRuntime.bytesSent[protocol] || 0) + queryString?.length || 0
-
-    const isValidJsonp = submitData.jsonp(
-      this.getScheme() + '://' + info.beacon + '/' + protocol + '/' + info.licenseKey + queryString,
-      jsonp
-    )
-    // Usually `drain` is invoked automatically after processing feature flags contained in the JSONP callback from
-    // ingest (see `activateFeatures`), so when JSONP cannot execute (as with module workers), we drain manually.
-    if (!isValidJsonp) drain(this.agentIdentifier, CONSTANTS.FEATURE_NAME)
+    harvester.send('rum', { qs: queryParameters, body }, { needResponse: true }, undefined, ({ responseText }) => {
```

Comment:
Do we need to handle retries / status codes under the new strat?

Reply:
We do not currently handle retries for RUM right now. I have moved this discussion to slack and will open tickets as necessary and the work will be completed separately.